### PR TITLE
Optionally show SniperCrosshair in Scope/Zoom

### PR DESCRIPTION
**Pull request by 0xE232FE**
You can now enable the Crosshair optionally in Zoom or Scope.

@danielkrupinski may I ask you to review this?

Thank you very much!

### CODE REVIEW

**Comment by 0xE232FE** [comment]
Disabled SniperCrosshair
SniperCross_Disabled


Enabled SniperCrosshair: Shows the additional Option
SniperCross_Enabled

Enable or Disable SniperCross when Scoped.
SniperCross_Enabled_With_Scope_Enabled

**Comment by AbdallahMehiz** [comment]
Thats a good one 👯‍♂

**Comment by xsincbt** [comment]
Gud job

**Comment by 0xE232FE** [comment]
Crash after Game Update
fixed vTable Index Update!

**Comment by 0xE232FE** [comment]
Fixed Headdot by draw less polygons!

**Comment by Bywnb** [comment]
isplayer is still 157 not 158

**Comment by 0xE232FE** [comment]
> isplayer is still 157 not 158
It is 157 from my vTable dump.

**Comment by nothanku** [comment]
Can you remove the blurry dot crosshair that's normally in the aug ang sg553? it looks kinda dumb with both of the crosshairs being there.

**Comment by 0xE232FE** [comment]
yes I can try
```
    IMaterial* __fastcall Hooks::FindMaterial(void* thisptr, void*, const char* pMaterialName, const char* pTextureGroupName, bool complain , const char* pComplainPrefix)
    {
    	static auto oFindMaterial = Hooks::MaterialSystem->GetOriginalFunction<FindMaterialFn>(84);
     
    	if( strcmp(pMaterialName, "dev/scope_bluroverlay") == 0)
    	{
    		//return other material instead, hehe
    		static auto ClearAsShit = oFindMaterial(thisptr, "dev/clearalpha", nullptr, complain, pComplainPrefix);
    		return ClearAsShit; //I wonder if there's any slight performance gain/loss from this
    	}
     
    	return oFindMaterial(thisptr, pMaterialName, pTextureGroupName, complain, pComplainPrefix);
    }
```

**Comment by nothanku** [comment]
Aight thanks. :) 
nice job anyway 👍

File: Osiris/GUI.cpp [hunk 1]
```diff
@@ -990,7 +990,11 @@ void GUI::renderMiscWindow(bool contentOnly) noexcept
     ImGui::Checkbox("Slowwalk", &config->misc.slowwalk);
     ImGui::SameLine();
     hotkey(config->misc.slowwalkKey);
-    ImGui::Checkbox("Sniper crosshair", &config->misc.sniperCrosshair);
```
Comment (playday3008) [inline Osiris/GUI.cpp]: Dont delete this

**Pull request closed without merging**